### DISCUSSION TITLE
Claude models showing when OpenAI is chosen

### DISCUSSION THREAD

**Discussion by hayschan** [danny-avila/LibreChat#3413] [Troubleshooting] [answered]
### What happened?

When selecting the OpenAI service in the interface, the model dropdown incorrectly displays models from Anthropic's Claude alongside OpenAI models. This leads to potential confusion as users might select a model that is not compatible with the chosen AI service.

#### Expected Behavior

When selecting OpenAI as the service, only OpenAI-specific models (like various versions of GPT) should be displayed in the model dropdown.

#### Actual Behavior

The model dropdown incorrectly displays models from both OpenAI and Anthropic when OpenAI is selected.

### Steps to Reproduce

1. Open the model selection dropdown.
2. Choose "OpenAI" from the AI service options.
3. Observe that the dropdown for models includes options such as "claude-3-haiku-20240307", which are specific to Anthropic.

### What browsers are you seeing the problem on?

Chrome

### Screenshots

![image](https://github.com/user-attachments/assets/e9095136-8a03-48b6-91a8-65575e7c713c)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hayschan** [comment]
### Version of the Docker image

![image](https://github.com/user-attachments/assets/9d51791c-5462-4183-bb80-b76a987e380a)


1. **MongoDB Container**
   - **Name:** chat-mongodb
   - **Version:** `63aaa9333a08deb8e13b74e5a59c50aaff412e1dd43ab94f78ea7ea31c37e043`

2. **LibreChat Main Application**
   - **Name:** LibreChat
   - **Image:** ghcr.io/danny-avila/librechat-dev:latest
   - **Version:** `dc774b5102b765b9b216917e75cb437b05627cfe5f842a43b390c1c77c682e84`

3. **RAG API Container**
   - **Name:** rag_api
   - **Image:** ghcr.io/danny-avila/librechat-rag-api-dev-lite:latest
   - **Version:** `14ec81c86e1197f0608108774d11c6dbcb00288d9521f1c2d3d4ff276640b822`

4. **Vector Database Container**
   - **Name:** vectordb
   - **Image:** ankane/pgvector:latest
   - **Version:** `440d04e15a4c72aaf0f731fe017c374b687a1386fc7611ab0c2ed4186994628a`

**Comment by danny-avila** [comment, marked as answer]
Are you using a reverse proxy for OpenAI? Because that list of models is populated from the API request for fetching models.

![image](https://github.com/user-attachments/assets/9ea57de1-aa65-4a9c-93ef-a5e265760f27)

**Reply by hayschan** [reply]
Yep, I am using a reverse proxy. Thanks for the help for the clarification. I thought it was a bug initially.
